sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -117,15 +117,14 @@ def get_responses():
   def _execute(self, task, request):
     try:
       response = task()
-    except Exception as e:  # pylint: disable=broad-except
-      traceback.print_exc(file=sys.stderr)
+    except Exception:  # pylint: disable=broad-except
+      traceback_string = traceback.format_exc()
+      print(traceback_string, file=sys.stderr)
       logging.error(
           'Error processing instruction %s. Original traceback is\n%s\n',
-          request.instruction_id,
-          traceback.format_exc(e),
```

Comment:
Note: `traceback.format_exc(e)` was mistakenly using the error as an argument, but this argument should actually be a numeric `limit` instead: https://docs.python.org/2/library/traceback.html#traceback.format_exc